services/viva-ms/lambdas/submitApplication.js

```diff
@@ -3,86 +3,132 @@ import AWS from 'aws-sdk';
 import to from 'await-to-js';
 
 import config from '../../../config';
+
+import * as dynamoDb from '../../../libs/dynamoDb';
+import log from '../../../libs/logs';
 import params from '../../../libs/params';
-import { putEvent } from '../../../libs/awsEventBridge';
+
+import putVivaMsEvent from '../helpers/putVivaMsEvent';
 import vivaAdapter from '../helpers/vivaAdapterRequestClient';
-import log from '../../../libs/logs';
 
 const VIVA_CASE_SSM_PARAMS = params.read(config.cases.providers.viva.envsKeyName);
+const VIVA_APPLICATION_RECEIVED = 'VIVA_APPLICATION_RECEIVED';
 
 const dynamoDbConverter = AWS.DynamoDB.Converter;
 
 export async function main(event, context) {
-  if (event.detail.dynamodb.NewImage === undefined) {
+  if (event.detail.dynamodb.NewImage == undefined) {
```

Comment:
Also what is the return type for a NewImage, then we could use strict equal since it's better than normal equal

Reply:
But will strict handle null?

Reply:
If we know the return type we can be more explicit in our handling. And in general i do think that it's better to use the Not `!` operator like `!event.detail.dynamodb.NewImage`. This will cover `undefined`, `null`, `""` and `false`.

Reply:
Will be fixed in future task